### DISCUSSION TITLE
Content Security Policy error when connecting to Vite dev server

### DISCUSSION THREAD

**Discussion by jasonfb** [ElMassimo/vite_ruby#397] [Q&A]
[troubleshooting section]: https://vite-ruby.netlify.app/guide/troubleshooting.html

- [x] I have tried upgrading by running `bundle update vite_ruby`.
- [x] I have read the __[troubleshooting section]__ before opening an issue.

### Description 📖
When loading either using two processes (`bin/vite dev` + `bin/rails s`) or using `bin/dev`, I get this console error:

Refused to connect to 'ws://localhost:3036/vite-dev/' because it violates the following Content Security Policy directive: "default-src 'self' https:". Note that 'connect-src' was not explicitly set, so 'default-src' is used as a fallback.

### Reproduction 🐞
_will update soon_

<details>
<summary>Vite Ruby Info</summary>
3.0

_Run `bin/rake vite:info` and provide the output:_

```
bin/vite present?: true
vite_ruby: 3.3.4
vite_rails: 3.0.15
rails: 7.0.6
node: v20.3.1
npm: 9.6.7
yarn: 1.22.19
pnpm: 
ruby: ruby 3.2.2 (2023-03-30 revision e51014f9c0) [arm64-darwin22]

installed packages:
momeas9@1.0.0 /Users/jason/Work/Momeas/Momeas9
├─┬ vite-plugin-ruby@3.2.2
│ └── vite@4.4.3 deduped
└── vite@4.4.3
```
</details>

### Logs 📜

<details>
<summary>Output</summary>

_Run `DEBUG=vite-plugin-ruby:* bin/vite dev` or `DEBUG=vite-plugin-ruby:* bin/vite build` and provide the output:_

```

```
</details>

**Comment by ElMassimo** [comment]
Hi Jason!

Check `config/initializers/content_security_policy.rb`, which should have [additional lines injected during installation](https://github.com/ElMassimo/vite_ruby/blob/main/vite_rails/lib/vite_rails/cli.rb#L41-L55).

If enabling CSP in development, you'll want to uncomment those lines to ensure the browser can connect to the Vite dev server directly (which is a different origin than the Rails server).

**Comment by jasonfb** [comment]
I've identified the problem. Annoyingly, this was caused by a rubocop autocorrect, which is hella annoying. I'm just documenting it for others.

please hang on while I am investigating.
